content/usgs-packages/Application_Intro.Rmd

```diff
@@ -32,14 +32,16 @@ set.seed(1)
 
 ## Lesson Summary
 
-TBD
+This lesson will revisit the packages taught in previous lessons and use some of their functions to create a common data analysis workflow in a modular and reproducible manner. Then, you can learn how a modular approach compares to traditional analyses.
+
+What is modularity and why is it more reproducible? Modularity is the idea that your workflow leverages other tools to do some of the more general tasks. You can think of it as a sort of "plug-and-play" scenario, where you are relying on functionality from another developer to perform pieces of your analysis. This type of workflow reduces some of the maintenance burden because you are pushing some of the responsibility for code accuracy and reliability onto the tools. This also means that you should choose your tools wisely and scrutinize the tools you plan to use in your analyses. 
 
 ## Lesson Objectives
 
 Use the packages and some of their functions to create a common data analysis workflow in a modular and reproducible manner.
 
 By the end of this lesson, the learner will be able to:
 
-1. Recognize similarities and differences in gsplot and base R plotting.
-2. Construct piped plotting calls.
-3. Create customized style files.
+1. Automate retrieval of data via NWIS, ScienceBase, and the GDP.
```

Comment:
because it's "via", maybe specify NWIS Web (the platform) in contrast to NWIS (the underlying data source). Might matter to some of our target audience